src/core/style/expressions/utils.js

```diff
@@ -1,6 +1,8 @@
 import { float, category } from '../functions';
 import Expression from './expression';
 
+export const DEFAULT = undefined;
```

Comment:
?

Reply:
@rochoa and I talked about adding a `DEFAULT` expression to be more explicit in the API to be used like: `torque($day, default, fade(5))`. I think we should have it at least on the string API, I don't have much opinion regarding the JS API